lib/presentation/my_listings/subviews/my_active_listing_dialog_view.dart

```diff
@@ -0,0 +1,377 @@
+                        const Icon(Icons.delete, size: 20, color: Colors.white),
+                        const SizedBox(width: 8),
+                        Text(
+                          I18n.of(
+                            context,
+                          ).myActiveListingDialogViewRemoveButton,
+                          style: const TextStyle(
+                            fontSize: 16,
+                            fontWeight: FontWeight.w500,
+                          ),
+                        ),
+                      ],
+                    ),
+                  ),
+                ),
+                const SizedBox(width: 16),
+                Expanded(
+                  child: OutlinedButton(
+                    onPressed:
+                        () => unawaited(
+                          context
+                              .read<MyActiveListingDialogCubit>()
+                              .reserveListing(
+                                widget.listing.id,
+                                _steamUrlController.text,
+                                _reservationNotesController.text,
+                              ),
+                        ),
+                    style: OutlinedButton.styleFrom(
+                      foregroundColor: AppColors.primary,
+                      padding: const EdgeInsets.symmetric(vertical: 16),
+                      shape: RoundedRectangleBorder(
+                        borderRadius: BorderRadius.circular(8),
+                      ),
+                      side: const BorderSide(
+                        color: AppColors.primary,
+                        width: 1,
+                      ),
+                      elevation: 0,
+                    ),
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        const Icon(
+                          Icons.event_available,
+                          size: 20,
+                          color: AppColors.primary,
+                        ),
+                        const SizedBox(width: 8),
+                        Text(
+                          I18n.of(
+                            context,
+                          ).myActiveListingDialogViewReserveButton,
+                          style: const TextStyle(
+                            fontSize: 16,
+                            fontWeight: FontWeight.w500,
+                          ),
+                        ),
+                      ],
+                    ),
+                  ),
+                ),
+              ],
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  @override
+  void dispose() {
+    _steamUrlController.dispose();
+    _reservationNotesController.dispose();
+    super.dispose();
+  }
+
+  @override
+  void initState() {
+    super.initState();
+
+    context.read<MyActiveListingDialogCubit>()
+      ..showToastError = (message) {
```

Comment:
use error message and display in the dialog instead of snackbar, as it is showing behind the dialog